cdk/domino_cdk/config/__init__.py

```diff
@@ -69,6 +70,7 @@ def config_template(
             "eks": {
                 "version": "1.19",
                 "private_api": private_api,
+                "secret_encrption_key_name": secrets_encryption_key_arn,
```

Comment:
```suggestion
                "secrets_encryption_key_arn": secrets_encryption_key_arn,
```

Reply:
Let's not be confusing. It is ARN, not a name.

Reply:
Sorry! I just meant to highlight the type in `secret_encryption_key_name`, totally miffed the copy-paste.

Reply:
Thanks!